equip.waste.yaml

```
@@ -0,0 +1,28 @@
+## Waste equipment definitions
+
+Bin
+ - Extend base type
+
+NAME: PWB                     ## Check for clashes
+  name: Portable Wheeled Bin
+  description: A lightweight portable bin for storing and transporting waste
```

```suggestion
BIN                
  name: Waste Bin
  description: A bin for storing and transporting waste
```
@TristanWebber a few comments
1. The main key for each YAML entry is the code.
2.  In this case, I suggest we genericize the bin object to "waste bin" rather than portable-wheeled bin to be more flexible in a wider variety of cases. Does that work for your use case?

---

Thanks - I have no issue with the generic approach.